mlflow/store/model_registry/file_store.py

```diff
@@ -573,7 +579,6 @@ def update_model_version(self, name, version, description):
         model_version.description = description
         model_version.last_updated_timestamp = updated_time
         self._save_model_version_as_meta_file(model_version)
-        self._update_registered_model_last_updated_time(name, updated_time)
```

Comment:
I was suprised by this, but, in `SqlAlchemyStore`, `descripiton` update doesn't update the last updated time.

Reply:
Maybe this is because changing `descripiton` is not a ciritical update.